rotkehlchen/api/v1/schemas.py

```diff
@@ -705,10 +704,26 @@ def make_history_event_filter(
         else:
             filter_query = HistoryEventFilterQuery.make(**common_arguments)
 
-        return {
+        base_filter = {
             'filter_query': filter_query,
-            'group_by_event_ids': data['group_by_event_ids'],
         }
+        base_filter.update(self.get_extra_fields(data))
+        return base_filter
+
+    def make_extra_common_arguments(self, data: dict[str, Any]) -> dict[str, Any]:
+        """
+        generates the extra fields to be included in the filter_query dictionary
+        """
+        return {
+            'limit': data['limit'],
+            'offset': data['offset'],
+        }
+
+    def get_extra_fields(self, data: dict[str, Any]) -> dict[str, Any]:
+        """
+        generates the extra fields to be included in the base_filter dictionary
+        """
```

Comment:
this naming is confusing. I was thinking that those were things that you add to the filter but instead are extra arguments that get returned after the processing.

```suggestion
    def validation_proxied_information(self, data: dict[str, Any]) -> dict[str, Any]:
        """
        extra fields that will be returned after validation
        """
```

Reply:
if you find a better name is good. What you had really confused me